jdk-tool-plugin/src/test/java/hudson/model/DownloadServiceTest.java

```diff
@@ -0,0 +1,42 @@
+package hudson.model;
+
+
+import hudson.tools.JDKInstaller;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import net.sf.json.JSONObject;
+import org.junit.Test;
+import org.junit.Rule;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * @author Kohsuke Kawaguchi
+ */
+public class DownloadServiceTest {
+
+    @Rule
+    public JenkinsRule r = new JenkinsRule();
```

Comment:
This test runs without the JenkinsRule, need to make sure it is unnecessary and clean it up.

Reply:
Should I add a comment that the test is extracted from `jenkins/test/src/test/java/hudson/model/DownloadServiceTest.java`?